include/base/OpenMCCellAverageProblem.h

```diff
@@ -360,9 +402,9 @@ class OpenMCCellAverageProblem : public OpenMCProblemBase
    * @param[in] score tally score
    * @return sum of the tally
    */
-  Real getCellTally(const unsigned int & var_num,
-                    const std::vector<xt::xtensor<double, 1>> & tally,
-                    const unsigned int & score);
+  // Real getCellTally(const unsigned int & var_num,
```

Comment:
If we don't need them anymore, let's remove the comments. Same applies to `getMeshTally`

Reply:
Oops, thanks for catching this! I didn't realize some remnants of the old tally system slipped through. I've also swept through and removed some other functions in `OpenMCCellAverageProblem` and `OpenMCProblemBase` which aren't used in the `Tallies` sub-block.